artemis/base_pytest.py

```diff
@@ -310,6 +309,40 @@ def put_data(data_type: str, file_suffix):
             logger.debug("Tyr response : {}".format(r.text))
             return True
 
+        def pause_tyr_beat():
+            """
+            Pause tyr_beat executable which lives in tyr_beat container.
+            Pauses all processes within tyr_beat container.
+            """
+            logger.debug("Stopping tyr_beat")
+            containers = [
+                x
+                for x in docker.DockerClient(version="auto").containers.list()
+                if "tyr_beat" in x.name
+            ]
+            if not containers:
+                logger.error("No Docker Container found for tyr_beat")
+                raise Exception("No Docker Container found for tyr_beat")
+            else:
+                containers[0].pause()
+
+        def unpause_tyr_beat():
+            """
+            Unpause tyr_beat executable which lives in tyr_beat container.
+            Unpauses all processes within tyr_beat container.
+            """
+            logger.debug("Starting tyr_beat")
+            containers = [
+                x
+                for x in docker.DockerClient(version="auto").containers.list()
+                if "tyr_beat" in x.name
+            ]
+            if not containers:
+                logger.error("No Docker Container found for tyr_beat")
+                raise Exception("No Docker Container found for tyr_beat")
+            else:
+                containers[0].unpause()
+
```

Comment:
```suggestion
```suggestion
for cn in contenairs:
         cn.unpause()
```
No ?

```

Reply:
I agree, we could do that, I used containers[0] because I knew there is only one tyr_beat container, but your solution is better !

Reply:
Done !